lib/print-diff-total.js

```diff
@@ -67,7 +67,9 @@ ${getPlural(filesDiff, 'item', 'items')}`;
 
   const result = `${sizeDiffPretty} (${gzipSizeDiffPretty})\
 ${filesDiff === 0 ? '' : `, ${filesDiffPretty}`}`;
-  const resultColor = sizeDiff > 0 || filesDiff > 0 ? 'red' : 'green';
+  const resultColor = sizeDiff > 0 || gzipSizeDiff > 0 || filesDiff > 0
```

Comment:
I think we should not count files number increasing as a negative result. 

We could have one big file, and we split it into two files, total size of which is less than the size of the original big file. In this case Harold will show it in red, won't it? But it's surely positive outcome.

What do you think? 🤔

Reply:
I think this is a good catch. So I removed `filesDiff > 0` check.